app/templates/order/_order_summary.hbs

```diff
@@ -0,0 +1,5 @@
+{{#unless cart.isEmpty}}
+  {{#unless isAppointment}}
+    {{small-cart-toggle-view}}
+  {{/unless}}
+{{/unless}}
```

Comment:
@akshaygwd , you can use `is-or` helper that we have to avoid nested conditionals here.

Reply:
@abhinav098 I want both conditions to be satisfied.

Reply:
in that case we have `is-and`.

Reply:
@akshaygwd in that case we have `is-and`.

Reply:
Hi, @abhinav098 Thanks for your suggestions. Appreciate it. 

You can ask @akshaygwd to correct this in another PR as it is not completely related to this PR and can be done separately as an improvement to our code.

It's really good to mention such things so that we can improve our codebase constantly. :)

@akshatgwd please keep a note of this suggestion and fix it whenever you get time.

Thanks.

Reply:
@namrataukirde yeah sure.